Nodejs/Product/Nodejs/Intellisense/VsProjectAnalyzer.cs

```diff
@@ -1075,41 +1075,6 @@ CollectingErrorSink errorSink
         ) {
             // There are some scenarios during ES6 Mode where a textview is still opened using NodeLS.
             // In these cases, we do not properly parse ES6 syntax, and therefore do not want to display errors. 
-            if (NodejsPackage.Instance.IntellisenseOptionsPage.AnalysisLevel == AnalysisLevel.Preview) {
-                return;
-            }
```

Comment:
nit: I would prefer to remove just this, and remove everything below in a separate, more focused change, when we remove the analyzer altogether